config/locales/en/documents/show.yml

```diff
@@ -32,6 +32,8 @@ en:
         status: Status
         updated_at: Last updated
         created_at: Created
+        created_by: Document created by
+        unknown_data: Unknown
```

Comment:
Tiny thing, but could we call this "unknown_creator", just so it's obvious where it's being shown?

Reply:
Sure - my thinking on this was it could be reused for other future metadata fields for which we don't have the data.